lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -415,6 +418,31 @@ export abstract class ClientApplication {
         });
     }
 
+    private trackPageVisibilityWithMeas():void {
+        if(!this.ssoSilentMeasurement && !this.acquireTokenByCodeAsyncMeasurement) return;
+        this.logger.info("Perf: Event- visibility change detected" );
```

Comment:
Throughout this method you're mixing vis and visibility, please adjust to the long form of Visible or Visibility throughout your change.

Reply:
fixed